packages/manager/modules/nutanix/src/node.class.js

```diff
@@ -31,14 +31,17 @@ export default class Node {
   }
 
   get isDeployed() {
-    if (this.os.startsWith(DEFAULT_OS_NODE_NUTANIX.split('.')[0])) {
-      return false;
+    if (
+      !this.os.startsWith(DEFAULT_OS_NODE_NUTANIX.split('.')[0]) &&
+      this.status === NODE_STATUS.UNKNOWN
```

Comment:
Checking the status value first would avoid unecessary splitting the string into array every time.
Niptick : all conditions could be grouped in only one return instruction
```suggestion
      this.status === NODE_STATUS.UNKNOWN &&
      !this.os.startsWith(DEFAULT_OS_NODE_NUTANIX.split('.')[0])
      
```